### ISSUE TITLE
two class segmentation (label 0,255 ),the iou =0 ,acc = nan, how to solve this problem?

### ISSUE THREAD

**Issue by chao1577135517** [open-mmlab/mmsegmentation#1609]
Problem: During training, the IOU=0, Acc=nan of the target class, the background is 100, and the loss cannot be reduced from 0.82-0.72 (LOSS uses diceloss), crossentropyloss, focalloss have been used, both are like this
Question: My labels are 0 and 255 (background, target), is it because 0 and 1 are not used?
![LN3AO}6W9TH0 F$6(VMV`)D](https://user-images.githubusercontent.com/90619392/169781467-6ebe872c-3b24-4b75-bf76-95cba35fe2f5.png)

**Comment by MengzhangLI** [comment]
Because `255` would be automatically ignored https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/custom.py#L89.

You can modify annotations to `1` and try again. Could follow DRIVE dataset:

https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/drive.py

https://github.com/open-mmlab/mmsegmentation/blob/master/configs/_base_/datasets/drive.py

**Comment by chao1577135517** [comment]
thanks，I will try it again

**Comment by earsonlau** [comment]
mask图里面的像素值：背景为0，前景为1.
